src/ISA/ISA/ArcTypes/ArcTable.fs

```diff
@@ -503,12 +503,17 @@ type ArcTable(name: string, headers: ResizeArray<CompositeHeader>, values: Syste
     /// <summary>
     /// This function can be used to join two arc tables.
     /// </summary>
+    /// <param name="index">If not set default to append. -1 will also append.</param>
     /// <param name="table">The table to join to this table.</param>
     /// <param name="joinOptions">Can add only headers, header with unitized cell information, headers with values.</param>
     /// <param name="forceReplace">if set to true will replace unique columns.</param>
-    member this.Join(table:ArcTable, ?joinOptions: TableJoinOptions, ?forceReplace: bool, ?SkipFillMissing) : unit =
+    member this.Join(table:ArcTable, ?index: int, ?joinOptions: TableJoinOptions, ?forceReplace: bool, ?skipFillMissing) : unit =
         let joinOptions = defaultArg joinOptions TableJoinOptions.Headers
         let forceReplace = defaultArg forceReplace false
+        let skipFillMissing = defaultArg skipFillMissing true
```

Comment:
Shouldn't the default value be `false`?

Reply:
Don't know, you had it as true:
![image](https://github.com/nfdi4plants/ARCtrl/assets/39732517/9b963f82-247d-44c6-a9d7-ffaf09024e4a)

Reply:
The `not` makes the difference here. You could read my code as follows: `Only if skipFillMissing is set to true, do not fill missing cells`

Reply:
The default case being `fill missing cells`

Reply:
The new one also has `true` with `not`?

Reply:
ah wait i think i get it. Because yours defaulted to None, it would not match